__tests__/endpoints.test.js

```diff
@@ -127,4 +127,54 @@ describe("/api/articles", () => {
         })
     })
     
+})
+
+describe("/api/articles/:article_id/comments", () => {
+    test("200: returns with and array of comments with correct keys for article with specific ID", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body}) => {
+            const comments = body[0]
+            const size = Object.keys(comments[0]).length;
+            expect(size).toBe(6)
```

Comment:
This size should be checking that the comments array has a specific length so we dont get a false passing test when given an empty array.